src/main/java/com/dao/MakePost.java

```diff
@@ -14,4 +14,8 @@ public static void readingPost()
 	{
 		System.out.println("Reading Post");
 	}
+	public static void deletePost()
+	{

```

Comment:
Before delete "validated user successfully" also print in  syso to better

Reply:
okay Mr.Raju Ganipireddi after modication i will update to you